src/vivarium_census_prl_synth_pop/tools/cli.py

```diff
@@ -66,14 +66,9 @@ def make_artifacts(
 
 @click.command()
 @click.argument("output_dir", type=click.Path(exists=True))
-@click.option(
-    "-V",
-    "--label-version",
-    type=str,
-    default=None,
-    show_default=True,
-    help="Provide a version number for final results. "
-    "Version should follow format of '#.#.#'.",
```

Comment:
Why did you remove this help text?

Reply:
I don't think arguments can have help text. But it could certainly go in the function's docstring

Reply:
Arguments can't. I can add a docstring and add it there.